### PR TITLE
Fixed metrics url; Collapse filter when science theme landing page open

### PR DESCRIPTION
**Pull request by chuanlin2018**
Fixed the following two issues:
1. The link to metrics page broken: fix: treat /pdr/metrics/mds2-XXXX and /pdr/metrics/ark:/88434/mds2-XXXX the same.
2. Need to collapse the filter panel when science theme page first open.

mds2-2157 can be used to test metrics fix.

### CODE REVIEW

File: angular/src/app/shared/searchfields-list/searchfields-list.service.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class SearchfieldsListService {
   constructor(private http: HttpClient,
     private appConfig: AppConfig,
     private cfg: AppConfig) {
-      this.RMMAPIURL = cfg.get("locations.mdService", "/unconfigured");
+      this.RMMAPIURL = cfg.get("API.mdSearch", "/unconfigured");
```
Comment (RayPlante) [inline angular/src/app/shared/searchfields-list/searchfields-list.service.ts:23]: This should be "APIs.mdSearch".  See [environment.ts](https://github.com/usnistgov/oar-pdr/blob/integration/angular/src/environments/environment.ts).

Comment (RayPlante) [reply angular/src/app/shared/searchfields-list/searchfields-list.service.ts:23]: I'll probably fix this.

**Review by RayPlante — CHANGES_REQUESTED** [review]
See inline comment regarding `mdSearch`; I'll probably fix this.

File: angular/src/app/shared/searchfields-list/searchfields-list.service.ts (hunk 1 above)
Comment (chuanlin2018) [reply angular/src/app/shared/searchfields-list/searchfields-list.service.ts:23]: Oh ya, I didn't test my last minute change... Go ahead fix it please.

**Pull request merged**